stanford/stage-ansible.cfg

```diff
@@ -1,13 +0,0 @@
-[defaults]
-extra_vars='key=deployment name=edx-stage group=edx-stage region=us-west-1'
-forks=10
-host_key_checking = False
-hostfile=../playbooks/ec2.py
-jinja2_extensions=jinja2.ext.do
-log_path=~/stage-ansible.log
-transport=ssh
-user=ubuntu
-
-[ssh_connection]
-ssh_args=-F stage-ssh-config -o ControlMaster=auto -o ControlPersist=60s -o ControlPath=/tmp/ansible-ssh-%h-%p-%r
```

Comment:
The `-F stage-ssh-config` flag is now conditionally set via our deploy script.
It currently exists in `stanford-dev`, but should soon be moved to `configuration`.